proposals/apple-mdm/Fleet-MDM-Setup.md

```diff
@@ -0,0 +1,152 @@
+# Setup
+
+The one-time setup consists of configuring:
```

Comment:
Since the certs expire yearly - part of this is one time but certificate updates need to happen regularly, so I would call it out here. I'll remove "one-time" from here + mention updates.